app/models/casa_case.rb

```diff
@@ -86,7 +86,7 @@ def update_cleaning_contact_types(args)
   def clear_court_dates
     update(court_date: nil,
            court_report_due_date: nil,
-           court_report_submitted: false)
+           court_report_submitted: false) if court_date.present? && court_date < Time.now
```

Comment:
Since `court_date` will only ever be `nil` or an actual date, you can simplify this by just checking for `if court_date` rather than using `if court_date.present?`. Also, please use `Time.current` instead of `Time.now`. Thoughtbot has a nice article about timezones. Here is [a link](https://thoughtbot.com/blog/its-about-time-zones#a-summary-of-do39s-and-don39ts-with-time-zones) to the summary.

Reply:
Ok thanks. will check it out